src/closure-externs.js

```diff
@@ -1093,17 +1096,76 @@ var FUNCTION_TABLE;
  */
 var MozBlobBuilder;
 /**
- * @suppress {undefinedVars}
+ * @suppress {duplicate, undefinedVars}
  */
 var GL;
 /**
- * @suppress {undefinedVars}
+ * @suppress {duplicate, undefinedVars}
+ */
+var WebGLClient;
+/**
+ * @suppress {duplicate, undefinedVars}
  */
 var SDL;
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var JSEvents;
 
 // Module loaders externs, for AMD etc.
 
 /**
  * @param {Function} wrapper
  */
 var define = function (wrapper) {};
+
+/**
+ * GL functions (https://github.com/kripken/emscripten/commit/97a464a654fdadf5dfb8aa082b48516e6bf8d402#commitcomment-25520648)
```

Comment:
These are rather for emulated fixed pipeline GL functions, instead of general GL functions, but probably distinguishing in this file won't matter.